src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/GetActivityResourceIntTest.kt

```diff
@@ -319,13 +267,13 @@ class GetActivityResourceIntTest : IntegrationTestBase() {
       }
 
       @Test
-      fun `should not include if prisoner active in different prison`() {
+      fun `should not include if course activity created by DPS`() {
         nomisDataBuilder.build {
           programService {
-            courseActivity = courseActivity(startDate = "$today", createdByDps = false)
+            courseActivity = courseActivity(startDate = "$today", createdByDps = true)
           }
           offender {
-            booking(agencyLocationId = "LEI") {
+            booking {
               courseAllocation(courseActivity = courseActivity, startDate = "$today")
             }
           }
```

Comment:
The diff is very hard to follow because lots of the tests look similar and git is easily confused.

In essence the changes are:
* remove any tests interested in prisoner status - we no longer require active prisoners allocated to activities
* add boundary tests to check allocation > 6 months old are ignored but others allow the activity migration